dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops

```diff
@@ -82,6 +82,11 @@ kafka.create.outgoing.retry.CWMRX1010W=CWMRX1010W: A Kafka outgoing connector fo
 kafka.create.outgoing.retry.CWMRX1010W.explanation=An error occurred when the Kafka connector was set up for an outgoing reactive messaging channel. This might be caused by a problem with the connector configuration, or by a problem resolving the Kafka broker hostname.
 kafka.create.outgoing.retry.CWMRX1010W.useraction=Review the error message, the messages.log file on the server, and FFDC logs to identify whether the problem was transient and was resolved by retrying the connector setup.
 
+# {0} = kafka record data, {1} = exception details
+# 'nack' is the name of a method and should not be translated 
+kafka.input.message.nacked.CWMRX1011E=CWMRX1011E: The nack method was called on a message created from a Kafka record, indicating that it was not be processed successfully. The reactive message stream will be shut down. The record was: {0}. The exception was: {1}.
+kafka.input.message.nacked.CWMRX1011E.explanation=A message sent by the kafka connector caused a failure somewhere in the application which resulted in the message being negatively acknowledged. As this message has not been successfully processed, the committed offset will not be advanced for the partition that the message came from and so no more messages can be processed from this partition.
+kafka.input.message.nacked.CWMRX1011E.useraction=Review the error message and the messages.log file on the server to identify the problem. If stopping processing of messages is not the desired behavior when message processing fails, add logic in your application which provides alternative handling of messages which are negatively-acknowledged.
```

Comment:
```suggestion
kafka.input.message.nacked.CWMRX1011E.useraction=Review the error message and the messages.log file on the server to identify the problem. To avoid message processing halting on a failure, implement logic in your application that provides alternative handling for messages that are negatively acknowledged.
```

Reply:
Update to message has been made and ready for re-review

Reply:
negatively-acknowledged is being corrected to negatively acknowledged - by acrolinx